volatility3/framework/symbols/windows/pdbutil.py

```diff
@@ -253,7 +253,8 @@ def pdbname_scan(cls,
                      pdb_names: List[bytes],
                      progress_callback: constants.ProgressCallback = None,
                      start: Optional[int] = None,
-                     end: Optional[int] = None) -> Generator[Dict[str, Optional[Union[bytes, str, int]]], None, None]:
+                     end: Optional[int] = None,
+                     maximum_invalid_count: int = 100) -> Generator[Dict[str, Optional[Union[bytes, str, int]]], None, None]:
```

Comment:
This seems alright, since in general we're not using the specific pages of the PE file itself, but now seems like a good time to document the parameters and what they do in the docstring please.  Happy to help define things if needed.

Reply:
Is 100 the lowest we can reasonably go?  How many pages are likely to have been paged out of the kernel (in fact, why are kernel pages paged out at all)?  Will this now hit ghost images and/or allow someone to partially load a different kernel early in memory to throw volatility off?  It's not a joke, we've had papers published about tricking volatility's automagic in the past, with tricks as similarly daft, so I'm keen to keep our tolerance to errors as low as possible.

Reply:
Some kernel pages can be invalid because they are paged out or discarded and marked invalid. In my image there were 25 invalid pages. Also, this function is layer agnostic and can be used by others (for non kernel layers) where paged can be swapped out.